src/modules/Transcript/index.tsx

```diff
@@ -605,9 +644,11 @@ function useAudioSync(
     Dispatch<SetStateAction<string | null>>,
     RefCallback<HTMLDivElement>,
     RefCallback<HTMLDivElement>,
-    Partial
+    Partial,
+    string | null
 ] {
     const [currentParagraph, setCurrentParagraph] = useState<string | null>(null);
+    const [currentParagraphTimestamp, setCurrentParagraphTimestamp] = useState<string | null>(null);
```

Comment:
Can you just use `currentParagraph` since that should be enough to get the timestamp instead of having to do a sep useState?